exporter/debugexporter/internal/normal/common.go

```diff
@@ -0,0 +1,20 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package normal
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+)
+
+// writeAttributes returns a slice of strings in the form "attrKey=attrValue"
+func writeAttributes(attributes pcommon.Map) (attributeStrings []string) {
+	attributes.Range(func(k string, v pcommon.Value) bool {
+		attribute := fmt.Sprintf("%s=%s", k, v.AsString())
```

Comment:
I think I could rewrite `writeAttributes` to take `buffer` and use `buffer.WriteString` instead of `fmt.Sprintf` if that helps? Perhaps as an follow-up improvement PR?

Reply:
Yeah that makes sense, fine for me to do this on a follow up PR